layers/+lang/javascript/packages.el

```diff
@@ -260,6 +261,38 @@
           "sR" 'spacemacs/skewer-eval-region-and-focus
           "ss" 'skewer-repl)))))
 
+(defun javascript/init-indium ()
+  (when (eq javascript-repl 'indium)
+    (use-package indium
+      :defer t
+      :init
+      (progn
+        (require 'indium)
+        (add-hook 'js2-mode-hook 'indium-interaction-mode)
+        ;; prefixes
+        (spacemacs/declare-prefix-for-mode 'indium-interaction-mode "mdi" "debugger")
+        (spacemacs/declare-prefix-for-mode 'indium-debugger-mode "mdd" "debugger")
+        ;; keybindings
+        (spacemacs/set-leader-keys-for-minor-mode 'indium-interaction-mode
```

Comment:
These are repl bindings and must be bound under the prefix `s` as to the conventions. Also there are certain standard bindings which must be set for a repl. Please have a look at how the `nodejs` repl is set up in this layer for a good example.